independent-projects/bootstrap/runner/src/main/java/io/quarkus/bootstrap/runner/QuarkusEntryPoint.java

```diff
@@ -13,7 +13,7 @@ public static void main(String... args) throws Exception {
         try (InputStream in = ClassLoader.getSystemClassLoader().getResourceAsStream(QUARKUS_APPLICATION_DAT)) {
             String path = QuarkusEntryPoint.class.getProtectionDomain().getCodeSource().getLocation().getPath();
             String decodedPath = URLDecoder.decode(path, "UTF-8");
-            SerializedApplication app = SerializedApplication.read(in, Paths.get(decodedPath).getParent().getParent());
+            SerializedApplication app = SerializedApplication.read(in, new File(decodedPath).toPath().getParent().getParent());
```

Comment:
Did this make a difference?

Reply:
This is the most important bit. new File and Paths.get() behave differently when passed a windows path